.github/workflows/pull-request-target.yml

```
@@ -0,0 +1,77 @@
+
+            ### CONTRIBUTING.md
+
+            Read our CONTRIBUTING.md file. Most importantly:
+
+            1. PRs with issues not assigned to you will be closed by the reviewer
+            1. Fix the first comment in the PR so that each issue listed automatically closes
+
+            ### Other
+
+            1. :dart: Please be considerate of our volunteers' time. Contacting the person who assigned the reviewers is not advised unless they ask for your input. Do not @ the person who did the assignment otherwise.
+            2. Read the CONTRIBUTING.md file make 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)